frontend/lib/analytics/google-analytics.tsx

```diff
@@ -283,14 +283,23 @@ type OutboundLinkProps = DetailedHTMLProps<
 > & {
   /** The "href" prop is required on outbound links, not optional. */
   href: string;
+  target: string;
+  rel: string;
 };
 
+const defaultOutboundLinkProps = {
+  target: "_blank",
+  rel: "noopener noreferrer"
+}
 /**
  * A react component that encapsulates a link to an external website,
  * which we want to track with analytics.
  */
 export function OutboundLink(props: OutboundLinkProps): JSX.Element {
-  const { onClick, ...otherProps } = props;
+  console.log("before default", props);
+  props = { ...props, ...defaultOutboundLinkProps,};
```

Comment:
This is good, but I think that because of how these two spreads are ordered, the `defaultOutboundLinkProps` will actually _override_ the passed-in ones, rather than fill them with default values in case they're missing.  You should be able to verify this by passing in different value for `target` or `rel` in an `<OutboundLink>`...  The rendered `<a>` might not have the values you think they should!